src/dashboard/ui/screens/draw_utils.rs

```diff
@@ -0,0 +1,110 @@
+use tui::widgets::Table;
+use tui::{
+    backend::Backend,
+    layout::{Constraint, Direction, Layout, Rect},
+    style::{Color, Modifier, Style},
+    text::{Span, Spans, Text},
+    widgets::{Block, Borders, Paragraph, Row},
+    Frame,
+};
+
+use crate::dashboard::models::StatefulTable;
+
+const DARK_FG_COLOR: Color = Color::White;
```

Comment:
Styles (light vs dark) are unused for now, let's remove them and just keep a single style.